pkgs/servers/mattermost/default.nix

```diff
@@ -34,7 +34,7 @@ buildGoModule rec {
 
   webapp = fetchurl {
     url = "https://releases.mattermost.com/${version}/mattermost-${version}-linux-amd64.tar.gz";
-    hash = "sha256-tgds8eTBeisuJcLgtx6zOiFUcVL1oU0LLbPqmh4SQUU=";
+    hash = "sha256-ZlvO/7kdMopIHBDdFp6wSQCR+NobGdDC6PcVd1iG16E=";
   };
 
   vendorHash = "sha256-TJCtgNf56A1U0EbV5gXjTro+YudVBRWiSZoBC3nJxnE=";
```

Comment:
Did this change if you blank it out? (Assumedly not)

Reply:
didn't

(… i would have noted that in the PR description, except I tweaked it, replaced it with nix's output, and then didn't notice it was the same hash as before :see_no_evil: )